### PR TITLE
DOC: update the pandas.Series.dt.is_quarter_end docstring

### PR DESCRIPTION
**Pull request by janelle**
Checklist for the pandas documentation sprint (ignore this if you are doing
an unrelated PR):

- [X] PR title is "DOC: update the <your-function-or-method> docstring"
- [X] The validation script passes: `scripts/validate_docstrings.py <your-function-or-method>`
- [X] The PEP8 style check passes: `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [X] The html version looks good: `python doc/make.py --single <your-function-or-method>`
- [X] It has been proofread on language by another sprint participant

Please include the output of the validation script below between the "```" ticks:

```
################################################################################
################# Docstring (pandas.Series.dt.is_quarter_end)  #################
################################################################################

Return a boolean indicating whether the date is the last day of a
quarter.

Returns
-------
is_quarter_end : Series of boolean.

See Also
--------
quarter : Return the quarter of the date.

is_quarter_start : Return a boolean indicating whether the date is the
    first day of the quarter.

Examples
--------
>>> df = pd.DataFrame({'dates': pd.date_range("2017-03-30",
...                   periods=4)})
>>> df.assign(quarter = df.dates.dt.quarter,
...          is_quarter_end = df.dates.dt.is_quarter_end)
       dates  quarter    is_quarter_end
0 2017-03-30        1             False
1 2017-03-31        1              True
2 2017-04-01        2             False
3 2017-04-02        2             False

################################################################################
################################## Validation ##################################
################################################################################

Errors found:
	No summary found (a short summary in a single line should be present at the beginning of the docstring)

```

If the validation script still gives errors, but you think there is a good reason
to deviate in this case (and there are certainly such cases), please state this
explicitly.

* No extended summary required.
* Short summary exceeds one line by one word, hope that is ok

### CODE REVIEW

File: pandas/core/indexes/datetimes.py [hunk 1]
```diff
@@ -1739,7 +1739,33 @@ def freq(self, value):
...
+        quarter : Return the quarter of the date.
+
+        is_quarter_start : Return a boolean indicating whether the date is the
+            first day of the quarter.
+
+        Examples
+        --------
+        >>> df = pd.DataFrame({'dates': pd.date_range("2017-03-30",
+        ...                   periods=4)})
+        >>> df.assign(quarter = df.dates.dt.quarter,
```
Comment (dukebody) [inline pandas/core/indexes/datetimes.py]: I'd say no spaces around the `=` sign here to follow pep8.

**Review by dukebody — APPROVED** [review]
Only minor pep8 formatting issue. I agree this is so simple an extended description is not really needed.

File: pandas/core/indexes/datetimes.py [hunk 2]
```diff
@@ -1739,7 +1739,33 @@ def freq(self, value):
     is_quarter_end = _field_accessor(
         'is_quarter_end',
         'is_quarter_end',
-        "Logical indicating if last day of quarter (defined by frequency)")
+        """
+        Return a boolean indicating whether the date is the last day of a
```
Comment (TomAugspurger) [inline pandas/core/indexes/datetimes.py]: See if you can reprhase to get this on a single line

File: pandas/core/indexes/datetimes.py [hunk 3]
```diff
@@ -1739,7 +1739,33 @@ def freq(self, value):
...
+        quarter.
+
+        Returns
+        -------
+        is_quarter_end : Series of boolean.
```
Comment (TomAugspurger) [inline pandas/core/indexes/datetimes.py]:
```
is_quarter_end : Series or DatetimeIndex
    The same type as the original data. Series will have the same
    name and index. DatetimeIndex will have the same name.
```

File: pandas/core/indexes/datetimes.py [hunk 4]
```diff
@@ -1739,7 +1739,33 @@ def freq(self, value):
...
+
+        See Also
+        --------
+        quarter : Return the quarter of the date.
```
Comment (TomAugspurger) [inline pandas/core/indexes/datetimes.py]: https://github.com/pandas-dev/pandas/pull/20275#discussion_r173651295

File: pandas/core/indexes/datetimes.py [hunk 5]
```diff
@@ -1739,7 +1739,33 @@ def freq(self, value):
...
+
+        is_quarter_start : Return a boolean indicating whether the date is the
+            first day of the quarter.
+
+        Examples
+        --------
+        >>> df = pd.DataFrame({'dates': pd.date_range("2017-03-30",
+        ...                   periods=4)})
```
Comment (TomAugspurger) [inline pandas/core/indexes/datetimes.py]: PEP: indent periods under the quote in `date_range(`

File: pandas/core/indexes/datetimes.py [hunk 6]
```diff
@@ -1739,7 +1739,33 @@ def freq(self, value):
...
+        >>> df.assign(quarter = df.dates.dt.quarter,
+        ...          is_quarter_end = df.dates.dt.is_quarter_end)
```
Comment (TomAugspurger) [inline pandas/core/indexes/datetimes.py]: PEP8: one more space

File: pandas/core/indexes/datetimes.py [hunk 7]
```diff
@@ -1739,7 +1739,33 @@ def freq(self, value):
...
+               dates  quarter    is_quarter_end
+        0 2017-03-30        1             False
+        1 2017-03-31        1              True
+        2 2017-04-01        2             False
+        3 2017-04-02        2             False
+        """)
```
Comment (TomAugspurger) [inline pandas/core/indexes/datetimes.py]: Example for DatetimeIndex

**Comment by TomAugspurger** [comment]
Thanks @janelle!

**Pull request merged**